autogpt/app.py

```diff
@@ -125,7 +125,7 @@ def execute_command(command_name: str, arguments):
                 google_result = google_official_search(arguments["input"])
             else:
                 google_result = google_search(arguments["input"])
-            safe_message = google_result.encode("utf-8", "ignore")
+            safe_message = [google_result_single.encode('utf-8', 'ignore') for google_result_single in google_result]
```

Comment:
Good catch, can we check that this is an instance of type list before iterating it though?

The result could be list[str] or str.

Reply:
You got it! Just updated to include this. Let me know if you'd like a specific string check and default 'else' as well.